cmd/siac/walletcmd.go

```diff
@@ -220,9 +223,31 @@ const unconfirmedTransactionTimestamp = ^uint64(0)
 // passwordPrompt securely reads a password from stdin.
 func passwordPrompt(prompt string) (string, error) {
 	fmt.Print(prompt)
-	pw, err := terminal.ReadPassword(int(syscall.Stdin))
-	fmt.Println()
-	return string(pw), err
+	if insecureInput {
+		prevState, err := term.MakeRaw(int(syscall.Stdin))
+		if err != nil {
+			return "", err
+		}
+		defer term.Restore(int(syscall.Stdin), prevState)
+		screen := struct {
+			io.Reader
+			io.Writer
+		}{
+			os.Stdin,
+			os.Stdout,
+		}
+		t := term.NewTerminal(screen, "")
+		pw, err := t.ReadLine()
+		if err != nil && err != io.EOF {
+			return pw, err
+		}
+		fmt.Println()
+		return string(pw), nil
```

Comment:
Surely this works just as well?
```go
pw, err := bufio.NewReader(os.Stdin).ReadString('\n')
````

Reply:
But does that handle the `\r` for windows users?

Reply:
I think if we do `strings.TrimSpace` (which we should probably do anyway) it'll be fine